webapp/channels/src/components/profile_popover/components/profile_popover_add_to_channel.tsx

```diff
@@ -52,42 +50,37 @@ const AddToChannel = ({
     if (!canManageAnyChannelMembersInCurrentTeam || !isInCurrentTeam) {
         return null;
     }
-    const addToChannelMessage = formatMessage({
-        id: 'user_profile.add_user_to_channel',
-        defaultMessage: 'Add to a Channel',
-    });
 
     return (
-        <OverlayTrigger
-            delayShow={Constants.OVERLAY_TIME_DELAY}
+        <WithTooltip
+            id='user_profile.add_user_to_channel.icon'
+            title={formatMessage({
+                id: 'user_profile.add_user_to_channel',
+                defaultMessage: 'Add to a Channel',
+            })}
             placement='top'
-            overlay={
-                <Tooltip id='addToChannelTooltip'>
-                    {addToChannelMessage}
-                </Tooltip>
-            }
         >
-            <div>
+            {/* This span is neccessary as tooltip is not able to pass trigger props to a custom component */}
+            <span>
```

Comment:
Does it matter that this is a span instead of a div now?

Reply:
No i doesnt, its just that div introduces display:block while span doesnt